crates/rooch-sequencer/src/actor/messages.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use coerce::actor::message::Message;
+use rooch_types::transaction::{authenticator::Authenticator, TypedTransaction};
+
+/// Transaction Sequence Message
+#[derive(Debug)]
+pub struct TransactionSequenceMessage {
+    pub tx: TypedTransaction,
+}
+
+#[derive(Debug)]
+pub struct TransactionSequenceResult {
+    /// The tx order
+    pub order: u128,
+    /// The tx order witness, usually it is the signature of the sequencer to prove the order
+    //TODO confirm the type.
+    pub order_witness: Authenticator,
```

Comment:
order_witness is for verifying tx & its order is correct in sequencer when someone wants to chanllege sequencer.

For normal tx request is just a signature for telling users that sequencer has confirmed that the tx has this order.

@jolestar @steelgeek091

Reply:
I put the signature here and called `order_witness`, the sequencer witness the tx order. In the challenge scenario, it should be `sequence_proof`, which proves the witness is correct.

Reply:
> I put the signature here and called `order_witness`, the sequencer witness the tx order. In the challenge scenario, it should be `sequence_proof`, which proves the witness is correct.

@jolestar 

membership/non-membership witness is proper noun for checking element existed or not in accumulator. So I think witness is not a good idea name here.

And for sequence_proof: Will it make people confused that this sequence and other sequence(it's widely used in other blockchain interface)?

Reply:
I change the `order_witness` to `tx_order_signature`.

Reply:
Upon reading numerous articles, I have observed the mix use of "witness" and "proof" which let me confused.

I ask the ChatGPT, and its answer is:
--------------------------------------------
In the accumulator scenario, the terms "membership witness" and "membership proof" are usually used to describe proving whether an element belongs to a specified accumulator. In this scenario, both terms can represent narrowing down the likelihood of "the element belongs to the accumulator".

Generally, "membership witness" refers to the proof that contains the specified element (as in Merkle tree), while "membership proof" refers to the complete proof path.

Therefore, if you only need to prove whether an element belongs to an accumulator, the term "membership witness" may be more intuitive. If you need the complete proof path, the term "membership proof" is more appropriate.